src/components/MoviesMenu/MoviesMenu.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "styles/theme";
+
+.subtitleAndMoviesLayout {
+  margin-top: 10px;
+  text-align: center;
+}
+
+.subtitle {
+  font-size:20px;
```

Comment:
I'd recommend against using font sizes that are not defined in your theme. Instead of:

```scss
 font-size: 20px;
```

Do something like:

```scss
  font-size: $sub-title;
```
Where `$sub-title` is defined in your `theme`.